Makefile

```diff
@@ -1,33 +1,46 @@
 DEFAULT_TARGET: build
+
+VERSION := 0.4.0
```

Comment:
How we would like to bump this value?

Reply:
IMO this is still to be discussed. I just wanted to do a first step in the direction of versioning all of our components in the same way.